Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -4147,9 +4147,13 @@ final class WorkspaceTests: XCTestCase {
         try testWithTemporaryDirectory { path in
             // Create a temporary package as a test case.
             let packagePath = path.appending(component: "MyPkg")
+            let xctestMinVersions = MinimumDeploymentTarget.default.xcTestMinimumDeploymentTargets
             let initPackage = try InitPackage(
                 name: packagePath.basename,
-                packageType: .executable,
+                options: .init(
+                    packageType: .executable,
+                    platforms: xctestMinVersions[.macOS].map{ .init(platform: .macOS, version: $0) }.map{ [$0] } ?? []
+                ),
```

Comment:
not sure about this change. the test is mimicking what users will get when they create a new package and if that emits warning / requires a min deployment target not to emit warning that is a not a good sign

Reply:
> Maybe, as long as XCTest is the only supported testing platform, SwiftPM could infer minimums for test targets based on the XCTest in the associated SDK?

I thought this was already the case?

Reply:
Possibly. I just checked and the warning seems to be gone.

Reply:
> Maybe, as long as XCTest is the only supported testing platform, SwiftPM could infer minimums for test targets based on the XCTest in the associated SDK?

> I thought this was already the case?

Perhaps this is the real bug.  I agree this wasn't the right fix but will use this PR to get to the bottom of this when I find the time.